duckduckgo/Android
# Update onboarding images to match new header

Task/Issue URL: https://app.asana.com/0/72649045549333/613439415394754
Tech Design URL: N/A

**Description**:
Updates the onboaridng image to match the redesigned header


**Steps to test this PR**:
1. Run a fresh installation of the app 
1. Ensure the onboarding images are updated

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
